modules/nf-core/seurat/multiseq/tests/main.nf.test

```diff
@@ -0,0 +1,130 @@
+// nf-core modules test seurat/multiseq
+nextflow_process {
+
+    name "Test Process SEURAT_MULTISEQ"
+    script "../main.nf"
+    process "SEURAT_MULTISEQ"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "seurat"
+    tag "seurat/multiseq"
+    tag "untar"
+
+    test("Standard_Multiome - 10x mtx") {
+        config "./nextflow.config"
+        setup {
+            run("UNTAR", alias:"UNTAR_HTO") {
+                script "modules/nf-core/untar/main.nf"
+                process {
+                """
+                input[0] = Channel.of([ [],file(params.modules_testdata_base_path + "/genomics/homo_sapiens/10xgenomics/cellranger/hashing_demultiplexing/hto.tar.gz", checkIfExists: true)   ])
+                """
+                }
+            } 
+            run("UNTAR", alias:"UNTAR_RNA") {
+                script "modules/nf-core/untar/main.nf"
+                process {
+                """
+                input[0] = Channel.of([ [],file(params.modules_testdata_base_path + "/genomics/homo_sapiens/10xgenomics/cellranger/hashing_demultiplexing/rna.tar.gz", checkIfExists: true)   ])
+                """
+                }
+            }
```

Comment:
You can move this setup block outside of this test (below the tags), then it will run only once.